src/actions/ScanActions.tsx

```diff
@@ -217,7 +217,10 @@ export function handleWalletUris(
         tokenId
       }
 
-      navigation.push('send2', { walletId: wallet.id, spendInfo })
+      // React navigation doesn't like passing non-serializable objects as params. Convert date to string first
+      // https://github.com/react-navigation/react-navigation/issues/7925
+      const isoExpireDate = parsedUri?.expireDate != null ? parsedUri.expireDate.toISOString() : undefined
```

Comment:
You can do `parsedUri?.expireDate?.toISOString()`. Stringing is fine, per our discussion the other day - it's not a public API, so go ahead and make the data easy to work with.

Reply:
done